packages/react-components/src/Modal/Header.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017-2021 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { HeaderProps } from './types';
+
+import React from 'react';
+import styled from 'styled-components';
+
+import Button from '../Button';
+
+export const Header: React.FC<HeaderProps> = ({ header, onClose }) => (
+  <ModalHeader className='header'>
+    {header && (
+      <h1>{header}</h1>
+    )}
+    <Button
+      dataTestId='close-modal'
+      icon='times'
+      onClick={onClose}
+    />
+  </ModalHeader>
+);
+
+const ModalHeader = styled.div`
```

Comment:
Although this is the way styled-components suggest, we really don't use it anywhere. It makes it incredibly difficult, imho, to reason about, hence it not being used.

Rather just `<div className='header'>` and what the export in React.memo(Header) with the styles. (As done elsewhere)

Reply:
✅